src/appshell/internal/applicationuiactions.cpp

```diff
@@ -136,10 +136,39 @@ const UiActionList ApplicationUiActions::m_actions = {
              QT_TRANSLATE_NOOP("action", "Toggle Playback Controls toolbar"),
              Checkable::Yes
              ),
+    UiAction("toggle-timeline",
+             mu::context::UiCtxNotationOpened,
+             QT_TRANSLATE_NOOP("action", "Timeline"),
+             QT_TRANSLATE_NOOP("action", "Toggle timeline"),
+             Checkable::Yes
+             ),
+    UiAction("synth-control",
```

Comment:
Why add this action?

Reply:
All actions that should be in the menu are added to the AppMenuModel. If the action is not registered, then an empty MenuItem is created and displayed in the view as a separator